USAGE.md

```diff
@@ -1,370 +1,707 @@
-The following is a table mapping of ATT&CK properties, the old ATT&CK MediaWiki names, and the new STIX properties. Some of these properties are standard STIX properties, while others were custom-created for compatibility with ATT&CK. These properties are accessed from STIX objects as JSON properties.
+The data in this repository is STIX2.0 and divided into three folders, one for each domain of ATT&CK. These domains generally follow the same format with a few departures. Domain differences will be noted in the relevant sections of this document. 
 
-### Migrating from MediaWiki
+STIX uses a different terminology than ATT&CK. The following table is a mapping of ATT&CK concepts to STIX 2.0 objects:
 
-### Common properties (on all objects)
-ATT&CK Property | ATT&CK MediaWiki | STIX Properties
---------------- | ---------------- | ---------------
-**Entry ID**    | `Has ID` | `external_references[i].external_id` where `external_references[i].source_name == "mitre-attack"`
-**Entry URL**   | `URL` | `external_references[i].url` where `external_references[i].source_name == "mitre-attack"`
-**Entry Title** | `Has display name` | `name`
-**Entry Text**  | `Has description` | `description`
-**Citation**    | `Citation reference` | `external_references`
-**Deprecated**  | `Deprecated` | `x_mitre_deprecated`
-**Revoked**     | `Not available via MediaWiki API` | `revoked`
-**Old ATT&CK ID** | `Not available via MediaWiki API` | `x_mitre_old_attack_id`
+| ATT&CK concept | STIX object type |
+|:------------|:----------|
+| [Matrix](#matrices)              | `x-mitre-matrix` |
+| [Tactic](#tactics)               | `x-mitre-tactic` |
+| [Technique](#techniques)         | `attack-pattern` |
+| [Sub-technique](#sub-techniques) | `attack-pattern` where `x_mitre_is_subtechnique = true` |
+| [Procedure](#procedures)         | `relationship` where `relationship_type = "uses"` and `target_ref` is an `attack-pattern` |
+| [Mitigation](#mitigations)       | `course-of-action` |
+| [Group](#groups)                 | `intrusion-set`  |
+| [Software](#software)            | `tool` or `malware` |
 
+Two additional object types are found in the ATT&CK catalog:
+ 
+| STIX object type | About |
+|:-----------------|:------|
+| `identity`       | Referenced in the `created_by_ref` of all objects to state that the MITRE Corporation created the object |
+| `marking-definition` | Referenced in the `object_marking_refs` of all objects to express the MITRE Corporation copyright |
+
+## Common departures from the STIX spec
+
+There are three general ways that ATT&CK departs from the stock STIX 2.0 format:
+- Custom object types. Object types prefixed with `x-mitre-`, e.g `x-mitre-matrix`, are custom STIX types not defined in the stock STIX spec. They follow the general [STIX Domain Object pattern](https://docs.oasis-open.org/cti/stix/v2.0/csprd01/part2-stix-objects/stix-v2.0-csprd01-part2-stix-objects.html#_Toc476230920) but describe concepts not covered by generic STIX2.0 types.
```

Comment:
I sort of answered my question in the previous comment, but now I'm slightly confused by the terminology. Since this is saying `x-mitre-` object types are custom STIX object types, but the table above that mapping ATT&CK concepts to STIX object types would lead me to believe `x-mitre-matrix` and `x-mitre-tactic` are part of the stock STIX 2.0 ? (This could also just be from my lack of familiarity with STIX and if the user is already expected to know it well and understand the distinction, then never mind!)

Reply:
Hmm that's a tough one. Custom STIX objects are still stix objects, they're just not part of the default STIX spec. I think users of the STIX spec would understand this since any `x_` field or `x-` object is custom, however not all readers are assumed to know about that feature of the STIX spec.

To clarify this I've updated the ATT&CK concepts to STIX type table I've added a new column marking where the types are custom, and provided links to the STIX spec where they're default types.